instrumentation/graphql/lib/opentelemetry/instrumentation/graphql/instrumentation.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module GraphQL
+      # The Instrumentation class contains logic to detect and install the GraphQL
+      # instrumentation
+      class Instrumentation < OpenTelemetry::Instrumentation::Base
+        install do |config|
+          require_dependencies
+          patch_graphql(config)
+        end
+
+        present do
+          defined?(::GraphQL)
+        end
+
+        private
+
+        def require_dependencies
+          require_relative 'patches/opentelemetry_graphql_tracing'
+        end
+
+        def patch_graphql(config = {})
+          config[:schemas].each { |s| patch_schema!(s) }
+        end
+
+        def target_version
+          Integration.version
+        end
```

Comment:
What is this for? I haven't found a reference to it (yet).

Reply:
👍 leftover from the port, will remove